infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/server/OpenShiftServerExposer.java

```diff
@@ -91,95 +87,22 @@
  */
 public class OpenShiftServerExposer extends KubernetesServerExposer<OpenShiftEnvironment> {
```

Comment:
This class is useless. The one reason to have it - defining docs for exposing external servers.
Maybe after adding java doc into `OpenShiftExternalServerExposer` this class can be removed.

Reply:
I'll try consolidating it with k8s infra, by using k8s server converter in OpenShiftEnvironmentProvisioner, and move the docs like you suggest.

Reply:
done